chain/beacon/node_test.go

```diff
@@ -602,8 +602,8 @@ func TestBeaconThreshold(t *testing.T) {
 }
 
 func TestProcessingPartialBeaconWithNonExistentIndexDoesntSegfault(t *testing.T) {
-	bls, _ := scheme.GetSchemeByID(scheme.DefaultSchemeID)
-	bt := NewBeaconTest(t, 3, 2, 30*time.Second, 0, bls, "default")
+	sch := crypto.SchemeFromName(scheme.DefaultSchemeID)
+	bt := NewBeaconTest(t, 3, 2, 30*time.Second, 0, *sch, "default")
```

Comment:
if we are providing the scheme by the env for test, perhaps the scheme logic could move instead `NewBeaconTest`